source/_integrations/zwave_js.markdown

```diff
@@ -20,56 +20,66 @@ ha_codeowners:
 
-### Supervisor managed installation
+3. A 16-byte **network key** used in order to connect securely to compatible devices. It is recommended that a network key is configured as security enabled devices may not function correctly if they are not added securely. You must provide this network key in the configuration part of the Z-Wave JS Server. For new installations, a default key will be auto generated for you.
 
-- The official Z-Wave JS add-on installed available from the add-on store.
+4. The Z-Wave JS integration in Home Assistant. This integration connects to the Z-Wave JS Server to retrieve the info from your Z-Wave network and turns it into Home Assistant devices and entities.
 
-### Core installation
+## Running the Z-Wave JS Server
 
-- The Z-Wave JS Server installed and running in your network.
-- We provide an official add-on which you can find in the Supervisor's add-on store.
-- Not running the Supervisor? The [Z-Wave JS 2 MQTT project](https://zwave-js.github.io/zwavejs2mqtt/#/getting-started/quick-start) also includes the Z-Wave JS Server (you can enable it in the settings) and they provide a Docker image.
+As stated in the above requirements, you need to run the Z-Wave JS Server somewhere in your network. There are multiple ways to run this server, we'll explain the most common ways below:
 
-### Hardware requirements
+### 1. The official Z-Wave JS add-on, available from the add-on store
 
-- [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the Z-Wave JS server is running.
+This is the recommended approach if you're running the default version of Home Assistant, including the supervisor and you do not require any advanced configuration/options but just want your Z-Wave devices available to control within Home Assistant.
 
-## Configuration
+### 2. The Z-Wave JS to MQTT add-on installed available from the community add-on store
+
+This is the recommended approach if you're running the default version of Home Assistant, including the supervisor and you like to have maximum control over your Z-Wave network and it's nodes. Despite what the name suggests, you can actually run this addon without MQTT enabled. In that case it will provide you with a full fledged, attractive and feature complete UI to manage your Z-Wave nodes and settings. The Z-Wave JS server is **included** in Zwave JS to MQTT, you enable it on the Z-Wave tab within the UI. If you install the add-on from the community repo, this will be done for you automatically.
```

Comment:
```suggestion
This is the recommended approach if you're running the default version of Home Assistant, including the Supervisor and you like to have maximum control over your Z-Wave network and its nodes.

Despite what the name suggests, you can actually run this addon without MQTT enabled. In that case, it will provide you with a full-fledged, attractive and feature-complete UI to manage your Z-Wave nodes and settings.

The Z-Wave JS server is **included** in Zwave JS to MQTT, you enable it on the Z-Wave tab within the UI. If you install the add-on from the community repo, this will be done for you automatically.
```